### DISCUSSION TITLE
New Firmware for RLC-511\522 and RLC-842

### DISCUSSION THREAD

**Discussion by ZdobnikovOB** [AT0myks/reolink-fw-archive#13] [Firmware sharing]
Fixed I-Frame switch button.
[IPC_51516M5M.757_21121512.RLC-522.OV05A10.5MP.AF.D7.REOLINK.pak](https://support.reolink.com/attachments/token/iSWMzBTCkhptip2tj4jFFUoXo/?name=IPC_51516M5M.757_21121512.RLC-522.OV05A10.5MP.AF.D7.REOLINK.pak)
[IPC_51516M5M.757_21121504.RLC-511.OV05A10.5MP.AF.REOLINK.pak](https://support.reolink.com/attachments/token/EsHhHRib4QhMDTE4SI1kINMw3/?name=IPC_51516M5M.757_21121504.RLC-511.OV05A10.5MP.AF.REOLINK.pak)

Fixed onvif Issue and I-Frame switch button.
[IPC_523D98MP.989_22051917.RLC-842A.IMX415.8MP.AF.REOLINK.pak](https://support.reolink.com/attachments/token/JH134V6um9pX3cfBwBhTz0l60/?name=IPC_523D98MP.989_22051917.RLC-842A.IMX415.8MP.AF.REOLINK.pak)

**Comment by AT0myks** [comment]
Thank you for sharing!

**Comment by TRusselo** [comment]
access denied.
